duckduckgo/iOS
# TouchId application lock

Reviewer: Caine or Francesco (whoever has time)
Asana: https://app.asana.com/0/361428290920652/372270002095768

**Description**:
Adds touchId support to the app

**NOTE:** I would like to squash rather than merge, feel free to review and then pass the PR back to me

**Steps to test this PR**:
1. Got to settings, enable touchID
2. Close the app and open it again. Note that authentication is required
3. Go to settings, disable touchID
4. Close the app and open it again. Note that authentication is no longer required

###### Reviewer Checklist:


###### PR DRI Checklist:
